backend/curriculum_tracking/api_views.py

```diff
@@ -572,3 +572,27 @@ def assign_user_as_reviewer(self, request, pk=None):
 
     # TODO: bulk set due dates
 
+
+class WorkshopAttendanceViewset(AuthMixin, viewsets.ModelViewSet):
+
+    serializer_class = serializers.WorkshopAttendanceSerializer
+    queryset = models.WorkshopAttendance.objects.order_by("pk")
+    filter_backends = [DjangoFilterBackend]
+    filterset_fields = ["attendee_user"]
+
+    def get_permissions(self):
+        if self.action == "retrieve":
+            permission_classes = [
+                curriculum_permissions.IsCardAssignee
```

Comment:
This is the permission class you are using.

```
class IsCardAssignee(BasePermission):
    def has_permission(self, request, view):
        user = request.user
        card = view.get_object()   #   a
        return user in card.assignees.all()   # b
```
In the line lebelled a, card would be made equal to the current workshop attendance instace.
In the line labelled b, we'll get an error because WorkshopAttendance does not have an attribute called assignee.

Here's what I need you to do:
1.write a unit test. In that test create a workshop attendance instance. try to GET that as a user who should not be allowed to get it. Expect a permission error
2. write a test where the user who attended the workshop GETs the workshop instance. Expect a sucessful response

When you wrrite those tests you will see that they both fail. 

Now make them pass